CONTRIBUTING.md

```diff
@@ -180,3 +180,21 @@ Use concise variable names.
 ### CSS names
 
 CSS class names, custom property names and keyframe names al must start with the prefix `denhaag-` followed by one or more [0-9a-z-]. This is also enforced by the linter via the `yarn lint:css` command.
+
+### Installing and testing packages locally
+We have tested this using a verdaccio local registry on the PWA master and development branches.
```

Comment:
Who is `we`? This should be specified or omitted from this text. Also, what is `this`? I would suggest changing the first sentence to "<whatever has been tested> has been tested using a Verdaccio"

Reply:
I've updated the text